.github/CONTRIBUTING.md

```diff
@@ -8,23 +8,26 @@ First off, thanks for taking the time to contribute! :clap::+1::tada:
 [Contributing Code](#contributing-code)
 
 [How Can I Contribute?](#how-can-i-contribute)
-  * [Reporting Bugs](#reporting-bugs)
-  * [Suggesting Enhancements](#suggesting-enhancements)
-  * [Submitting pull requests](#pull-requests)
+
+* [Reporting Bugs](#reporting-bugs)
+* [Suggesting Enhancements](#suggesting-enhancements)
+* [Submitting pull requests](#pull-requests)
 
 [Style guides](#style-guides)
-  * [git commit messages](#git-commit-messages)
-  * [documentation style guide](#documentation-style-guide)
-  * [python style guide](#python-style-guide)
+
+* [git commit messages](#git-commit-messages)
+* [documentation style guide](#documentation-style-guide)
+* [python style guide](#python-style-guide)
 
 ### Contributing Code
 
 1. Fork the KwanLab Autometa repository. You can do this [here](https://github.com/KwanLab/Autometa).
 2. <details><summary>Add the KwanLab as the upstream remote.</summary><code>
     git remote add upstream https://github.com/KwanLab/Autometa.git</code></details>
 3. Update your local repository with the most recent updates from the Kwan Lab.
-    - <details><summary>Update your local repository.</summary><code>git pull upstream master</code></details>
-    - <details><summary>Push changes to update your remote repository.</summary><code>git push origin master</code></details>
+  - <details><summary>Update your local repository.</summary><code>git pull upstream master</code></details>
+  - <details><summary>Push changes to update your remote repository.</summary><code>git push origin master</code></details>
+  - <details><summary>Ensure pre-commit hooks are installed.</summary><code>conda install pre-commit -y; pre-commit install</code></details>
```

Comment:
This command didn't work for me. I had to do:

```
conda install -c conda-forge pre-commit; pre-commit install
```

Reply:
Did you run this from this branch? `pre-commit install` will look for the `pre-commit-config.yaml` file.

Reply:
No I did not, but the point is it did not work without specifying conda-forge.

Reply:
Which command did not work for you?

1. `conda install -c conda-forge pre-commit`
2. `pre-commit install`

Reply:
You have put `conda install pre-commit`. That didn't work for me, I had to use `conda install -c conda-forge`. You have missed out `-c conda forge` in the file.

Reply:
Ahh, I see. Fixed now. 👍 🔧 📝